alcs-api/src/main.ts

```diff
@@ -72,6 +73,19 @@ function registerPipes(app: NestFastifyApplication) {
   );
 }
 
+async function registerMultiPart(app: NestFastifyApplication) {
+  await app.register(fastifyMultipart, {
+    limits: {
```

Comment:
any file type limits?

Reply:
I wasn't given any, what are you thinking?

Reply:
pdf, doc, docx, xls, xlsx, png, jpg, jpeg and something for geo data

Reply:
Just checked and there is no setting for Fastify Multipart to limit file types, this is something we will have to do on our side.